packages/code-du-travail-api/src/routes/enterprises/enterprises.elastic.ts

```diff
@@ -0,0 +1,32 @@
+import { SOURCES } from "@socialgouv/cdtn-sources";
+
+export interface Agreement {
+  num: number;
+  slug: string;
+}
+
+export interface SearchAgreementsBody {
+  _source: string[];
+  from: number;
+  query: unknown;
+  size: number;
+}
```

Comment:
Il vaudrait mieux des types que des interfaces car on étend pas

Reply:
Si je converti en type j'ai l'erreur suivante :
"ESLint: Use an `interface` instead of a `type`.(@typescript-eslint/consistent-type-definitions)"